tests/main/interfaces-time-control/task.yaml

```diff
@@ -16,11 +16,14 @@ prepare: |
 
 restore: |
     # Restore the initial rtc configuration
+    snap connect test-snapd-timedate-control-consumer:netlink-audit
+    snap connect test-snapd-timedate-control-consumer:time-control
     if [ -f rtc.txt ]; then
-        timedatectl set-local-rtc "$(cat rtc.txt)"
+        test-snapd-timedate-control-consumer.timedatectl-time set-local-rtc "$(cat rtc.txt)"
```

Comment:
I'm quite confused why we're using the snap we did the testing with here but not the host's timedatectl?

Reply:
I'll revert that, it was just for testing